src/h3lib/lib/h3Index.c

```diff
@@ -480,6 +481,31 @@ H3Index _h3RotatePent60ccw(H3Index h) {
     return h;
 }
 
+/**
+ * Rotate an H3Index 60 degrees clockwise about a pentagonal center.
+ * @param h The H3Index.
+ */
+H3Index _h3RotatePent60cw(H3Index h) {
```

Comment:
Nit: Is `60cw` right here, or are we really rotating 72 degrees? Is there a more semantic way to express this, like `_h3RotatePentSideCW`?

Reply:
This parallels the existing `_h3RotatePent60ccw`, which is a special case of `_h3Rotate60ccw`/`_h3Rotate60cw`. I think it could be argued we're rotating 60 degrees, but also skipping over part of the coordinate space.